workflows/mag.nf

```diff
@@ -288,10 +291,12 @@ workflow MAG {
         ch_versions = ch_versions.mix(BOWTIE2_PHIX_REMOVAL_ALIGN.out.versions.first())
     }
 
-    FASTQC_TRIMMED (
-        ch_short_reads
-    )
-    ch_versions = ch_versions.mix(FASTQC_TRIMMED.out.versions)
+    if (!(params.keep_phix && params.skip_clipping && !(params.host_genome || params.host_fasta))) {
+        FASTQC_TRIMMED (
+            ch_short_reads
+        )
+        ch_versions = ch_versions.mix(FASTQC_TRIMMED.out.versions)
+    }
```

Comment:
I don't fully understand the logic of this, why you would still want run FASTQC after removing phix? Just to get the stats of number reads left or something?

Reply:
Fair point, I think fine is either way. I'll let @skrakau  and/or @d4straub to decide for a majority vote ;)

Reply:
Fine to me either way.

Reply:
Keep it as it is then!